stataggregator/countaggregatorconfig.go

```diff
@@ -0,0 +1,54 @@
+package stataggregator
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	stat "github.com/asecurityteam/component-stat"
+	"github.com/rs/xstats"
+)
+
+// CountAggregatorConfig is for configuration a CountAggregator.
+type CountAggregatorConfig struct {
+	FlushInterval time.Duration `description:"Frequencing of aggregating metrics before sending."`
```

Comment:
Typo: I believe you want "Frequency" here instead of "Frequencing"

Reply:
yep, fixed